packages/compiler/src/server/tspconfig/completion.ts

```diff
@@ -40,30 +66,32 @@ export async function provideTspconfigCompletionItems(
         nodePath[0] === "emit" &&
         targetType === "arr-item")
     ) {
-      const emitters = await emitterProvider.listEmitters(tspConfigFile);
+      const libs = await emitterProvider.listLibraries(tspConfigFile);
       const items: CompletionItem[] = [];
-      for (const [name, pkg] of Object.entries(emitters)) {
+      for (const [name, pkg] of Object.entries(libs)) {
         if (!siblings.includes(name)) {
+          // Generate new text
+          const newText = getNewTextValue(sourceType, name);
+
           const item: CompletionItem = {
             label: name,
             kind: CompletionItemKind.Field,
             documentation: (await pkg.getPackageJsonData())?.description ?? `Emitter from ${name}`,
-            insertText: `"${name}"`,
+            textEdit: getNewTextAndPosition(newText, source, tspConfigPosition),
```

Comment:
why insertText not work here?

Reply:
In order to solve this issue: emitter option auto complete while inside "" will add extra ""`, the getNewTextValue method was originally intended for this purpose